apps/ecommerce/frontend/src/components/Config/ConfigPage.tsx

```diff
@@ -71,22 +70,20 @@ const ConfigPage = () => {
     })();
   }, [sdk, getConfig]);
 
-  const handleCredentialCheck = async (baseUrl: string) => {
-    const url = new URL(`${baseUrl}/healthcheck`);
-    fetchWithSignedRequest(url, sdk.ids.app, cma, sdk, 'POST', {
-      'X-Contentful-Data-Provider': sdk.parameters.instance.provider,
-      'x-contentful-shopify-shop': parameters.shopName,
-      'x-contentful-shopify-token': parameters.storefrontAccessToken,
+  const checkCredentials = async () => {
+    const url = new URL(`${config.backendApiUrl}/api/credentials`);
+    fetchWithSignedRequest(url, sdk.ids.app, cma, sdk, 'GET', {
+      // we only pass the installation parameters to the backend via headers here because they are not yet persisted
+      'x-data-provider-parameters': JSON.stringify(parameters),
     })
-      .then((res) => {
-        if (res.status !== 200) {
-          const error = `Error: ${res.status} - ${res.statusText}`;
-          throw new Error(error);
+      .then((res) => res.json())
+      .then((data) => {
```

Comment:
Style thought: I see you're just being consistent with what's already here, and we don't really have a style rule for this or anything, but it seems like we prefer await/async these days vs. then promise chains.